src/EntryPoint/RestEntryPoints.ts

```diff
@@ -51,11 +51,39 @@ export class RestEntryPoints implements EntryPoints {
     });
   }
 
-  public async entrypoints(projectId: string): Promise<EntryPoint[]> {
-    const res = await this.client.get(
-      `/api/v2/projects/${projectId}/entry-points`
-    );
+  public async entrypoints(
+    projectId: string,
+    limit: number = 10
+  ): Promise<EntryPoint[]> {
+    let l = limit;
+    const batchSize = 50;
```

Comment:
Extract this constant to be a member of the class or an instance

Reply:
done, thanks!